server/src/services/person.service.ts

```diff
@@ -298,8 +299,8 @@ export class PersonService {
     }
 
     if (force) {
-      await this.deleteAllPeople();
-      await this.repository.deleteAllFaces();
+      await this.repository.deleteAllFaces({ sourceType: null });
```

Comment:
Is it critical that the implementation here be in a transaction in the repository? It seems like it would be better to test separate actions here in the service instead. So like 
```ts
expect(faceMock.deleteByAssetId).toHaveBeenCalledWith(SourceType.MACHINE_LEARNING)
expect(faceMock.create).toHaveBeenCalledWith([...])
```

Reply:
I think it's best for delete + create to be atomic where possible.

Reply:
Hi @jrasm91, @mertalev should I change something here?